source/systems/pan/index.rst

```diff
@@ -16,12 +16,608 @@ Intel processors, from Sandy Bridge to Coffee Lake, each has
+specific host, hit control-C and enter the host name.
+
+C-shell Setup
+-------------
+
+In your ~/.cshrc and ~/.login, use the C-shell "if" statements below to
+separate settings for analysis logins from settings for your workstation:
+
+.. code-block:: shell
+
+   if ( `gfdl_platform` == hpcs-csc ) then
+      # settings for analysis logins
+   endif
+   if ( `gfdl_platform` == desktop ) then
+      # settings for workstations
+   endif
+
+
+C-shell keybinds for analysis
+-----------------------------
+
+If you use a C-shell (csh or tcsh) you may find that your key bindings on
+analysis are broken. e.g., the delete key produces a '~' instead of deleting a
+character. To fix this, create a ~/.bindings file with the following
+contents (mirroring the contents present in /etc/inputrc on the analysis nodes,
+but converting them for csh.) Add a line to your `~/.cshrc` file under the
+hpss-csc section from the C-shell Setup guidance above telling it to source
+~/.bindings
+
+.. code-block:: shell
+
+   ## for linux console and RH/Debian xterm
+   bindkey '^[[1~' beginning-of-line
+   bindkey '^[[4~' end-of-line
+   bindkey '^[[5~' history-search-backward
+   bindkey '^[[6~' history-search-forward
+   bindkey '^[[3~' delete-char
+   bindkey '^[[2~' quoted-insert
+   bindkey '^[[5C' forward-word
+   bindkey '^[[5D' backward-word
+   bindkey '^[[1;5C' forward-word
+   bindkey '^[[1;5D' backward-word
+   # for rxvt
+   bindkey '^[[8~' end-of-line
+   bindkey '^[Oc' forward-word
+   bindkey '^[Od' backward-word
+   # for non RH/Debian xterm, can't hurt for RH/DEbian xterm
+   bindkey '^[OH' beginning-of-line
+   bindkey '^[OF' end-of-line
+   # for freebsd console
+   bindkey '^[[H' beginning-of-line
+   bindkey '^[[F' end-of-line
+
+
+xterm and  gnome-terminal
+-------------------------
+
+After login to the analysis cluster, you may invoke:
+
+.. code-block:: shell
+
+   xterm &
+   gnome-terminal &
+
+These child sessions inherit the parent's scratch directory ($TMPDIR). By
+default, gnome-terminal does not read your .login file.
+
+Logging into the PP nodes
+-------------------------
+
+Sometimes, for debugging or other purposes, it can be useful to log into the pp
+nodes and run diagnostics / check on system resources. To do so, ssh in from
+any of the analysis nodes (workstations don't work):
+
+.. code-block:: shell
+
+  >ssh ${pp_nodename}.princeton.rdhpcs.noaa.gov
+
+where ${pp_nodename} is one of the pp nodes (i.e. pp212, pp301). The same
+method will work from analysis (not from the workstations) to log into an
+analysis node; simply swap out ${pp_nodename} for ${analysis_nodename}
+
+File Systems
+============
+
+In each GFDL filesystem, you have your own directory to work in. Two names
+point to this directory: your username (First.Last) and your "userdir"
+(initials).
+
+For example, user First.Last may access his home directory as either:
+
+.. code-block:: shell
+
+   /home/First.Last
+   /home/fl
```

Comment:
I don't think this is correct.  The short naming convention is always 3 characters, with a middle initial included.